Dockerfile

```diff
@@ -122,7 +122,9 @@ RUN make argocd-all
 ARG BUILD_ALL_CLIS=true
 RUN if [ "$BUILD_ALL_CLIS" = "true" ] ; then \
     make BIN_NAME=argocd-darwin-amd64 GOOS=darwin argocd-all && \
-    make BIN_NAME=argocd-windows-amd64.exe GOOS=windows argocd-all \
+    make BIN_NAME=argocd-windows-amd64.exe GOOS=windows argocd-all && \
+    make BIN_NAME=argocd-linux-amd64 GOOS=linux GOARCH=amd64 argocd-all && \
+    make BIN_NAME=argocd-linux-arm64 GOOS=linux GOARCH=arm64 argocd-all \
```

Comment:
Correct me if I am wrong. But, this change will create an additional binary (85MB est) which is duplicate of `argocd` binary built [here](https://github.com/argoproj/argo-cd/pull/5284/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L120)